### ISSUE TITLE
二分类问题 - sigmoid - softmax

### ISSUE THREAD

**Issue by junxnone** [junxnone/tech-io#577]
# Reference
- [ ] [Keras - Softmax 与 Sigmoid 二分类问题分析](https://www.aiuai.cn/aifarm679.html)
